user_tools/src/spark_rapids_pytools/resources/dev/log4j.properties

```diff
@@ -0,0 +1,20 @@
+# Set root logger level to INFO and its only appender to file
+log4j.rootLogger=INFO, FILE, stderr
+
+# File appender configuration
+log4j.appender.FILE=org.apache.log4j.FileAppender
+log4j.appender.file.append=false
+log4j.appender.FILE.File=./log4j.log
+log4j.appender.FILE.layout=org.apache.log4j.PatternLayout
+log4j.appender.FILE.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %t %p %c{1}:%L - %m%n
+
+# Console appender configuration for stderr
+log4j.appender.stderr=org.apache.log4j.ConsoleAppender
+log4j.appender.stderr.target=System.err
+log4j.appender.stderr.layout=org.apache.log4j.PatternLayout
+log4j.appender.stderr.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss.SSS} %t %p %c{1}: %m%n
+
+# Logging level for package
```

Comment:
Not certain if we need this section. We can remove it if we want to capture INFO level logging for all packages.